ui/pages/confirm-approve/confirm-approve.js

```diff
@@ -83,6 +79,12 @@ export default function ConfirmApprove() {
 
   const previousTokenAmount = useRef(tokenAmount);
 
+  const {
+    approveTransaction,
+    showCustomizeGasPopover,
+    closeCustomizeGasPopover,
+  } = useApproveTransaction(transaction);
```

Comment:
It looks like the `transaction` param passed to `useApproveTransaction` is not used (within `useApproveTransaction`)